recipes/r-ngsplot/build.sh

```diff
@@ -4,8 +4,7 @@ mkdir -p $outdir
 mkdir -p $PREFIX/bin
 cp -R * $outdir 
 #Set up links for 
-for f in $outdir/*; do
-    ln -s $outdir/* $PREFIX/bin
+for f in ${outdir}/bin/*; do
+    ln -s ${f} ${PREFIX}/bin
     fbname=$(basename "$f")
```

Comment:
Can be removed now.

Reply:
I don't get it right now - why?

Reply:
Sorry, I've been imprecise. Just meant the `fbname=$(basename "$f")` part :wink: